platformtour/static/js/src/platformtour_studio.js

```diff
@@ -0,0 +1,63 @@
+function PlatformTourStudioUI(runtime, element) {
+    'use strict';
+
+    var $ = window.$;
+    var $element = $(element);
+    var buttonSave = $element.find('.save-button');
+    var buttonCancel = $element.find('.cancel-button');
+    var url = runtime.handlerUrl(element, 'studio_view_save');
+
+    var customSteps = $("#platformtour_studio_custom_steps");
```

Comment:
Can you just give the element a classname like custom-steps and then lookup with:

`$element.find('.custom-steps')` ?

Reply:
Studio uses IDs on all entry fields. So I don't think changing this is necessary? See `platformtour_studio.html`, the first fields are literal copies of what Studio generated with the `studio_editable` mixin...